src/Sulu/Component/Util/SuluVersionPass.php

```diff
@@ -59,15 +60,41 @@ private function getSuluVersion($dir)
     }
 
     /**
-     * Read composer.json file and return version of app.
+     * Get version from composer file.
      *
      * @param string $dir
      *
-     * @return string
+     * @return string|null
      */
     private function getAppVersion($dir)
     {
-        $version = '_._._';
+        $key = 'version';
+        return $this->getValueByKeyFromComposerJson($dir, $key);
+    }
+
+    /**
+     * Get title from composer file.
+     *
+     * @param string $dir
+     *
+     * @return string|null
+     */
+    private function getTitle($dir)
+    {
+        $key = 'title';
```

Comment:
Title does not exist in composer this will throw the following error:

> The property title is not defined and the definition does not allow additional properties

Only description exists else it need to be in `extra` part of the composer.json 

/cc @danrot